specification/electronic-prescription-service-api.yaml

```diff
@@ -132,26 +132,32 @@ info:
-    | Endpoint              | HTTP code | Response format                                                                                                                                             |
-    | --------------------- | --------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------- |
-    | POST /$prepare        | 200       | [See /$prepare response body](https://digital.nhs.uk/developer/api-catalogue/electronic-prescription-service-fhir#api-Prescribing-preparePrescription)      |
-    | POST /$process-message| 200       | [See /$process-message response body](https://digital.nhs.uk/developer/api-catalogue/electronic-prescription-service-fhir#api-Prescribing-sendPrescription) |
+    
+    | Endpoint               | HTTP code | Response format                                                                                                                                             |
+    | ---------------------- | --------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------- |
+    | POST /$prepare         | 200       | [See /$prepare response body](https://digital.nhs.uk/developer/api-catalogue/electronic-prescription-service-fhir#api-Prescribing-preparePrescription)      |
+    | POST /$process-message | 200       | [See /$process-message response body](https://digital.nhs.uk/developer/api-catalogue/electronic-prescription-service-fhir#api-Prescribing-sendPrescription) |
+    
     ### 4xx responses
     **Note: This section is under development and subject to change. All Spine responses are returned with 200 HTTP codes until a Spine translation service is built. This does not affect the sandbox environment.**
+    
     | Endpoint              | HTTP code | Response format                           | Environment      | Cause                 |
     | --------------------- | --------- | ----------------------------------------- | ---------------- | --------------------- |
     | Any                   | 400       | See "Unsupported operation" example below | Sandbox          | Unsupported operation |
     | Any                   | 408       | Returns original request data             | Integration test | Request timeout       |
+    
     #### Validation errors
     The following errors may be returned by any endpoint in any environment and will all return a 400 HTTP status code.
-    | Issue severity | Issue type | Error code             | Cause                 |
-    | -------------- | ---------- | ---------------------- | --------------------- |
+    
+    | Issue severity | Issue type | Error code             | Cause                                                           |
+    | -------------- | ---------- | ---------------------- | ---------------------                                           |
```

Comment:
Just a guess but it might fall over here from missing dashes --- ... |

Reply:
added in JIC -- but the dashes are correct for the 2xx and 4xx tables which are also not formatting correctly :weary: